src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,25 @@
-// Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+
+type Props = {
+  todo: Todo,
+};
+
+export const TodoInfo: React.FC<Props> = (props) => {
+  const { todo: { title, completed, user } } = props;
```

Comment:
```suggestion
export const TodoInfo: React.FC<Props> = ({ todo }) => {
  const { title, completed, user } = todo;
```

I hope it is the easier way